dbms/src/Storages/DeltaMerge/Index/RoughCheck.h

```diff
@@ -56,6 +56,27 @@ inline RSResult checkEqual(const Field & v, const DataTypePtr & type, const T &
         return None;
 }
 
+template <typename T>
+inline RSResult checkIn(const std::vector<Field> & values, const DataTypePtr & type, const T & min, const T & max, const UInt8 has_null)
+{
+    RSResult result = None;
+    for (const auto & v : values)
+    {
+        if (result == All)
+            break;
+        if (v.isNull())
+        {
+            if (has_null)
```

Comment:
The expression `null in (null)` is also null. So can we just set `result = result || None` without check the value of `has_null`?

Reply:
you are right, fixed.